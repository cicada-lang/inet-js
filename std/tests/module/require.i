require "../../datatype/Nat.i"

eval @inspect(@run(@inspect(add(one(), zero()))))
