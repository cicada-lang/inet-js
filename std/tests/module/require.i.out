netFromPort (add₁₀)-result
  (add₁₀)-target!value-(add1₂₁)
  (add₁₀)-addend value-(zero₂₀)
  (add1₂₁)-prev value-(zero₁₉)
end
netFromPort (add1₂₂)-value!
  (add1₂₂)-prev value-(zero₂₀)
end
